targets/stm32l4x6.ld

```diff
@@ -0,0 +1,11 @@
+
+MEMORY
+{
+  FLASH_TEXT (rx) : ORIGIN = 0x08000000, LENGTH = 1024K
+  RAM (xrw)       : ORIGIN = 0x20000000, LENGTH = 96K
+  RAM2 (xrw)      : ORIGIN = 0x10000000, LENGTH = 32K
+}
+
+_stack_size = 4K;
+
+INCLUDE "targets/arm.ld"
```

Comment:
Please add LF at the end of the file. I think it's better to keep it consistent with other files.

Reply:
done